src/styles/blocks/_catalog.scss

```
@@ -0,0 +1,37 @@
+.catalog {
+  display: grid;
+  justify-content: center;
+  gap: 48px;
+
+  @media (max-width: 488px) {
+    grid-template-columns: $card-width;
+  }
+
+  @media (min-width: 488px) {
+    & {
+      grid-template-columns: repeat(2, $card-width);
+    }
+  }
+
+  @media (min-width: 768px) {
+    & {
+      grid-template-columns: repeat(3, $card-width);
+    }
+  }
+
+  @media (min-width: 1024px) {
+    & {
+      grid-template-columns: repeat(4, $card-width);
+    }
+  }
```

No needs to use `&` when you do n't add any specification like this
```
.catalog {
  &__list {
    ...
  }
}
```

```suggestion
  @media (min-width: 488px) {
    grid-template-columns: repeat(2, $card-width);
  }

  @media (min-width: 768px) {
    grid-template-columns: repeat(3, $card-width);
  }

  @media (min-width: 1024px) {
    grid-template-columns: repeat(4, $card-width);
  }
```